Code/Tools/AssetProcessor/native/ui/MainWindow.ui

```diff
@@ -1464,18 +1464,18 @@
              </layout>
             </widget>
            </widget>
-          </item>
+          </item>dis
```

Comment:
What happened here?

Reply:
Good catch, not sure where that came from (assume some accidental keystrokes during shutdown of VisualStudio and committing the file). Will fix.